README.md

```diff
@@ -1 +1,66 @@
-# m3-math-magicians
\ No newline at end of file
+# Getting Started with Create React App
+
```

Comment:
- [ ] (Optional) Kindly rename the header of your `README.md` file. Ideally, it should be the same name as that of your application. 👍

Reply:
### All comments have been uptated.

 _I really appreciate the call_ ✔